sources/europe/pl/Geoportal2Ewidencjagruntów.geojson

```diff
@@ -0,0 +1,433 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Geoportal2-PL-plot_boundaries_WMS",
+        "name": "Geoportal 2: Granice działek",
+        "i18n": true,
+        "description": "Usługa Krajowa Integracja Ewidencji Gruntów jest usługą zbiorczą prezentacji danych ewidencyjnych pochodzących bezpośrednio z jednostek szczebla powiatowego. Usługa zawiera jedynie dane tych jednostek, które dysponują usługą WMS o odpowiednich parametrach i zdecydowały się włączyć swoją usługę WMS do usługi zbiorczej KrajowaIntegracjaEwidencjiGruntow.",
+        "type": "wms",
+        "url": "https://integracja.gugik.gov.pl/cgi-bin/KrajowaIntegracjaEwidencjiGruntow?LAYERS=dzialki&STYLES=default&FORMAT=image/png&TRANSPARENT=TRUE&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
+        "country_code": "PL",
+        "available_projections": [
+            "EPSG:2176",
+            "EPSG:2177",
+            "EPSG:2178",
+            "EPSG:2179",
+            "EPSG:2180",
+            "EPSG:3857",
+            "EPSG:4326"
+        ],
+        "attribution": {
+            "text": "Główny Urząd Geodezji i Kartografii",
+            "url": "https://www.geoportal.gov.pl/",
+            "required": true
+        },
+        "icon": "https://wiki.openstreetmap.org/w/images/2/25/Geoportal-josm.png",
+        "terms-of-use-text": "https://geoportal.gov.pl/web/guest/regulamin",
+        "max_zoom": 23,
+        "license_url": "https://wiki.openstreetmap.org/wiki/Geoportal.gov.pl",
+        "custom-http-headers": {
+            "header-name": "User-Agent",
+            "header-value": "Mozilla/5.0 (JOSM)"
```

Comment:
was this necessary?

Reply:
It makes mapping forests, housing areas and fences easer. And it was requested be members of the Polish community, we already have it for JOSM

Reply:
For some reason it is necessary for the other Geoportal maps to work ones JOSM imports them from here